atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/schema/stream/generated/DeletingStreamStore.java

```diff
@@ -0,0 +1,17 @@
+package com.palantir.atlasdb.schema.stream.generated;
+
+import java.util.Set;
+
+import com.palantir.atlasdb.transaction.api.Transaction;
+
+public class DeletingStreamStore {
+    private final StreamTestStreamStore underlyingStreamStore;
+
+    public DeletingStreamStore(StreamTestStreamStore underlyingStreamStore) {
+        this.underlyingStreamStore = underlyingStreamStore;
+    }
+
+    public void deleteStreams(Transaction t, Set<Long> streamIds) {
+        underlyingStreamStore.deleteStreams(t, streamIds);
```

Comment:
`StreamTestStreamStore.deleteStreams` is package private, this exposes it. Let me know if you have other thoughts about how to change the package structure.